tests/pipelines/consisid/test_consisid.py

```diff
@@ -357,5 +358,5 @@ def test_consisid(self):
         video = videos[0]
         expected_video = torch.randn(1, 16, 480, 720, 3).numpy()
 
-        max_diff = numpy_cosine_similarity_distance(video, expected_video)
+        max_diff = numpy_cosine_similarity_distance(video.cpu(), expected_video)
```

Comment:
need to cpu, since `expected_video` is in cpu,  `numpy_cosine_similarity_distance` only supports 2 tensors in same device.